libs/organization/src/lib/organization/pages/organization-pending/organization-pending.component.html

```diff
@@ -5,9 +5,20 @@ <h1>Last Step</h1>
   <mat-card>
     <mat-card-content fxLayout="column" fxLayoutAlign="space-around center">
       <ng-container *ngIf="orgActive$ | async as active">
-        <p class="mat-body-2">
+        <ng-container *ngIf="app === 'festival' "class="mat-body-2">
+          <p>
+            The Archipel Market team will come back to you
```

Comment:
keep use `{{ app | appName }}` in case we change the appName
```suggestion
            The {{ app | appName }} team will come back to you
```